docs/contributor/draft/rate-limit-design.md

```diff
@@ -0,0 +1,80 @@
+# Rate Limit Proposal
+
+## Spec 
+
+| field                                            | type        | description                                                     | required                      |
+|--------------------------------------------------|-------------|-----------------------------------------------------------------|-------------------------------|
+| wokloadSelector                                  | object      | when skipped, it's applied for the whole namespace workloads, if empty and in istio-system ns, applies to all workloads in all namespaces | no                            |
+| workloadSelector.labels                          | object      | allows to select given workload by a label selector                   | no                            |
```

Comment:
Why are we not reusing the expressive Istio documentation?
I think the type is not object but map<string, string>.
```suggestion
| workloadSelector.labels                          | map<string, string>      | One or more labels that indicate a specific set of pods/VMs on which the configuration should be applied. The scope of label search is restricted to the configuration namespace in which the the resource is present.                  | no                            |
```